### PR TITLE
granatier: init at 19.12.3

### PR DESCRIPTION
**Pull request by freezeboy**
###### Motivation for this change

make granatier application available in kdeApplications

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/kde/granatier.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{ mkDerivation, lib
+, libkdegames, extra-cmake-modules
+, kdeclarative, knewstuff
+}:
+
+mkDerivation {
+  name = "granatier";
```
Comment (endgame) [inline pkgs/applications/kde/granatier.nix:7]: This seems to match the pattern set by other KDE applications, but I would have expected to see `version` defined somewhere. But I'm not that familiar with the KDE infrastructure.

File: pkgs/applications/kde/granatier.nix [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  meta = with lib; { maintainers = with maintainers; [ freezeboy ]; };
```
Comment (endgame) [inline pkgs/applications/kde/granatier.nix:8]: Please set `description` and `license` fields.

**Review by endgame — CHANGES_REQUESTED** [review]
Tested to build and run with `nix-build -A kdeApplications.granatier https://github.com/freezeboy/nixpkgs/archive/49d8c701135792fe257c199a2dd2a8486b4a5e64.tar.gz`

Some metadata is missing, please fix.

File: pkgs/applications/kde/granatier.nix (hunk 1 above)
Comment (freezeboy) [reply pkgs/applications/kde/granatier.nix:7]: true, there is no version attribute in the derivation, but the name of the generated package still has one, I didn't look how they did it,

don't know how to properly fix this

File: pkgs/applications/kde/granatier.nix (hunk 2 above)
Comment (freezeboy) [reply pkgs/applications/kde/granatier.nix]: Done

File: pkgs/applications/kde/granatier.nix (hunk 1 above)
Comment (endgame) [reply pkgs/applications/kde/granatier.nix:7]: It is possibly some KDE infrastructure magic, and you seem to be following common practice for that section of nixpkgs.

**Comment by endgame** [comment]
Please squash all your changes into a single commit with message `granatier: init at 19.12.3`. Otherwise, I'm happy with this. Actually getting it merged is above my pay grade, sorry.

**Comment by freezeboy** [comment]
Great, I think the squash can be done while merging the PR

File: pkgs/applications/kde/granatier.nix (hunk 1 above)
Comment (ttuegel) [reply pkgs/applications/kde/granatier.nix:7]: The version is set for all `kdeApplications` at once, because they are upgraded as a unit.

**Review by ttuegel — APPROVED** [review]

**Comment by ttuegel** [comment]
@ofborg build granatier

**Comment by freezeboy** [comment]
@ttuegel is it ok for merging ?

**Pull request merged**